apps/jobs/views.py

```diff
@@ -224,8 +238,78 @@ def leaderboard(request, challenge_phase_split_id):
 
     leaderboard_labels = challenge_phase_split.leaderboard.schema['labels']
     for item in distinct_sorted_leaderboard_data:
-        item['result'] = [item['result'][index.lower()] for index in leaderboard_labels]
+        item['result'] = [item['result'][index.lower()]
+                          for index in leaderboard_labels]
 
-    paginator, result_page = paginated_queryset(distinct_sorted_leaderboard_data, request)
+    paginator, result_page = paginated_queryset(
+        distinct_sorted_leaderboard_data, request)
     response_data = result_page
     return paginator.get_paginated_response(response_data)
+
+
+@throttle_classes([UserRateThrottle])
+@api_view(['GET'])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_remaining_submissions(request, challenge_phase_pk, participant_team_pk, challenge_pk):
+    challenge = get_challenge_object(challenge_pk)
```

Comment:
@RishabhJain2018 : I feel that the below parts need some more thoughts and probably rewrite from you. Can you please go through it and see if it adheres to the concerned task